clientv3/options.go

```diff
@@ -47,7 +47,7 @@ var (
 	// client-side streaming retry limit, only applied to requests where server responds with
 	// a error code clearly indicating it was unable to process the request such as codes.Unavailable.
 	// If set to 0, retry is disabled.
-	defaultStreamMaxRetries = uint(^uint(0)) // max uint
+	defaultStreamMaxRetries = uint(0) // max uint
```

Comment:
this does not seem to be right. we still want all 1's in the binary representation.

Reply:
My bad, I will fix it. Thanks @xiang90 for catching it.